source/Constant.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  MAX_INTEGER: 4294967295,
+  MIN_INTEGER: -4294967295,
+  CHARACTERS: 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789',
```

Comment:
Can possibly remove this file
* Since `CHARACTERS` is used in only one file it can be moved to the file/fn where it's used
* With the use of JS native limits